sdk/operationalinsights/Azure.ResourceManager.OperationalInsights/CHANGELOG.md

```diff
@@ -1,14 +1,6 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+## 1.0.0 (2022-11-29)
```

Comment:
Please follow https://github.com/Azure/azure-sdk-for-net/pull/32035 to update the changelog.md  and readme.md

Reply:
Updated